src/natcap/invest/utils.py

```diff
@@ -175,23 +161,22 @@ def prepare_workspace(
     with capture_gdal_logging(), log_to_file(logfile,
                                              exclude_threads=exclude_threads,
                                              logging_level=logging_level):
-        with sandbox_tempdir(dir=workspace):
```

Comment:
This context manager wasn't doing anything here. It yields a temporary directory path, which we weren't assigning to a variable or using in any way.